src/stan_inference.jl

```diff
@@ -50,7 +50,8 @@ function stan_inference(prob::DiffEqBase.DEProblem,t,data,priors = nothing;alg=:
                             num_samples=1000, num_warmup=1000, reltol=1e-3,
                             abstol=1e-6, maxiter=Int(1e5),likelihood=Normal,
                             vars=(StanODEData(),InverseGamma(3,3)),nchains=1,
-                            sample_u0 = false, save_idxs = nothing, diffeq_string = nothing)
+                            sample_u0 = false, save_idxs = nothing, diffeq_string = nothing, printsummary = true)
+  save_idxs !== nothing && length(save_idxs) == 1 ? save_idxs = save_idxs[1] : save_idxs = save_idxs
```

Comment:
?

Reply:
Needed to handle single valued arrays https://github.com/SciML/DiffEqBayes.jl/pull/145/files/27e634d965df3634e668f0f9e6d6787b3c135e44#diff-53adadc90178dc5b19921124ef5eefa3R81 here